scripts/geome_things.py

```diff
@@ -344,6 +344,73 @@ def reloadRecords(ctx, status_code):
     finally:
         session.close()
 
+@main.command("populate_isb_core_solr")
```

Comment:
This should be mostly identical to the code in `sesar_things`, I don't think you need to review it too closely.